src/raven/client.clj

```diff
@@ -137,21 +137,22 @@
 
 (defn merged-payload
   "Return a payload map depending on the type of the event."
-  [ev ts pid uuid localhost]
+  [event ts pid uuid localhost]
   (merge (default-payload ts localhost)
          (cond
-           (map? ev)       ev
-           (exception? ev) (exception->ev ev)
-           :else           {:message (str ev)})
+           (map? event)       event
+           (exception? event) (exception->ev event)
+           :else              {:message (str event)})
          {:event_id uuid
           :project  pid}))
 
 (defn add-breadcrumbs-to-payload
-  [payload]
-  (merge payload
-         (cond
-           (empty? @@breadcrumbs)  {}
-           :else   {:breadcrumbs {:values @@breadcrumbs}})))
+  [context payload]
+  (let [breadcrumbs-list (:breadcrumbs context)]
```

Comment:
This is a good candidate for `cond->` it should be self explanatory except for `seq` which is a weird clojure idiom for `not-empty?`

```clojure
(cond-> payload (seq breadcrumbs-list) (assoc :breadcrumbs {:values breadcrumbs-list}))
```

Reply:
Done.

Thanks, that's a much nicer form indeed :)